docs/framework_utils.rst

```diff
@@ -74,8 +74,16 @@ Common Filters
 .. automodule:: redbot.core.utils.common_filters
     :members:
 
+==========
 Utility UI
 ==========
 
 .. automodule:: redbot.core.utils.views
     :members:
+
+========
```

Comment:
It doesn't seem to make sense for these two to have a higher heading than the other utility modules, I think.
```suggestion
Utility UI
==========

.. automodule:: redbot.core.utils.views
    :members:

```